OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
Hmm, I see.  That's even more broken than I was remembering/assuming.  Lets just fix the bug, and then deal with the balance consequences later.  The community lived through the missile behaviour fix, and can live through this too.